lib/commands.js

```diff
@@ -34,6 +39,50 @@ function noPermissions (message) {
   }).catch(() => message.channel.send(errorMessage + ' here'))
 }
 
+function searchMainGithubIssues (value) {
+  return fetch(`https://api.github.com/search/issues?q=repo:runelite/runelite+${value}`, {
+    headers: {
+      Authorization: `token ${config.githubToken}`
+    }
+  }).then(res => res.json())
+}
+
+function addCommentToGithubIssue (id, message) {
+  searchMainGithubIssues(id).then(body => {
+    if (body.items.length === 0) {
+      return
+    }
+
+    const commentsURL = body.items[0].comments_url
+
+    if (!commentsURL) {
+      return
+    }
+
+    return fetch(commentsURL, {
+      method: 'POST',
+      headers: {
+        'Authorization': `token ${config.githubToken}`,
+        'User-Agent': 'RuneLite-Bot',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        body: message
+      })
```

Comment:
Given you have Content-Type set to JSON, do you need a JSON.stringify call? (I don't know the answer myself)